server/publications.js

```diff
@@ -18,6 +18,10 @@ ContextBlocks._ensureIndex({
 });
 
 ContextBlocks._ensureIndex({
+  videoMarker: 1
```

Comment:
I don't think this is necessary, since you aren't doing any server-side lookups based on videoMarker. (and probably don't need to either)